internal/ui/qml-new/Controls/BackButton.qml

```diff
@@ -0,0 +1,45 @@
+import QtQuick 2.7
+import QtQuick.Controls 2.1
+import "../Style"
+
+Item {
+    id: root
+
+    implicitHeight: 32
```

Comment:
Should we add a DPI factor for sizes?

Reply:
isn't it automatically fixed by Qt nowadays? As pixels don't represent points but are adapting depending on the display DPI?